src/Icon/Icon.tsx

```diff
@@ -173,6 +173,67 @@ export const Icon: FC<Props> = ({
         />
       </svg>
     )}
+
+    {// REFUND (Box with arrow facing left)
+    render === 'refund' && (
+      <svg width="100%" height="100%" viewBox="0 0 32 32" fill="none">
+        {/* tslint:disable-next-line:max-line-length */}
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
```

Comment:
I believe React wouldn't be happy with **SVG** attributes not being in camel case